en/docs/deploy/security/enable-hsts.md

```diff
@@ -62,22 +61,22 @@ Similar to the management console, check whether the `HttpHeaderSecurityFilter`
 
 ## Enable for Jaggery applications
```

Comment:
This section talks about the old dashboard

Reply:
Commented it out. Please confirm if this section is only used for the dashboard functionality in which case we might have to hide the whole section.